src/ui-kit/form-controls/autocomplete-multiselect/autocomplete-cache.ts

```diff
@@ -92,15 +93,16 @@ export class AutocompleteCache {
   }
 
   public static arraysEqual (arr1, arr2) {
-    if (arr1.length !== arr2.length) {
-      return false;
-    }
-    for (let i = arr1.length; i--; ) {
-      if (arr1[i] !== arr2[i]) {
-        return false;
-      }
-    }
-    return true;
+    // if (arr1.length !== arr2.length) {
+    //   return false;
+    // }
+    // for (let i = arr1.length; i--; ) {
+    //   if (arr1[i] !== arr2[i]) {
+    //     return false;
+    //   }
+    // }
+    // return true;
+    return isEqual(arr1, arr2);
```

Comment:
could we use src/ui-kit/utilities/are-equal/are-equal.ts instead?

Reply:
yeah that's an alternative. The areEqual method doesn't work when you're comparing 2 arrays directly, and have to loop through them, that's why I went for the other option. But if dependency is a concern then using areEqual is better.